### PR TITLE
Fix build errors and CI

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

This PR fixes the build error which is currently blocking CI.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by chingucoding** [comment]
Looks like CI is messed a bit since the packaging changes were merged, looking into fixing this.

**Comment by chingucoding** [comment]
CI should be fixed now.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -36,13 +36,14 @@ steps:
...
       inputs:
-          platform: $(buildPlatform)
-          solution: "$(solutionGallery)"
-          configuration: "$(buildConfiguration)"
-          msbuildArgs: '/p:PublishReadyToRun=false /p:AppxBundlePlatforms="$(buildPlatform)" /p:AppxPackageDir="$(appxPackageDir)" /p:AppxBundle=Always /p:UapAppxPackageBuildMode=SideloadOnly /p:AppxPackageSigningEnabled=true'
+        command: 'publish'
+        publishWebProjects: false
+        projects: '$(solutionGallery)'
+        arguments: '/p:PublishReadyToRun=false /p:AppxBundlePlatforms="x64" /p:AppxPackageDir="D:\a\1\a\AppxPackages\\" /p:AppxBundle=Always /p:UapAppxPackageBuildMode=SideloadOnly /p:AppxPackageSigningEnabled=true /p:platform="x64" /p:configuration="Debug" /p:PublishProfile="./WinUIGallery/Properties/PublishProfiles/win10-x64.pubxml"'
+        zipAfterPublish: false
```
Comment (karkarl) [inline azure-pipelines.yml:45]: Should the variables AppxBundlePlatforms, AppxPackageDir, platform and configuration not be hardcoded?

Comment (chingucoding) [reply azure-pipelines.yml:45]: Do you mean inside the CSProj file or as variables for the CI pipeline? I think hardcoding them inside the project file is not ideal given the fact that people not using those values will probably not have the best experience working with the repo.

Comment (chingucoding) [reply azure-pipelines.yml:45]: I've updated the CI command now and its using the build platform parameter now.

**Review by karkarl — APPROVED** [review]

**Pull request merged**